### ISSUE TITLE
Ideenfindung

### ISSUE THREAD

**Issue by t-karcher** [t-karcher/Spiel-zum-Familientag#2]
@ChrisOKay:  Ich überlege, ob wir nicht ein Spiel zum Familientag im September basteln könnten. Mögliche Themen:
- Stammbaum
- Familiengeschichte
  - Brauerei
- Treysa
- ...

Vom Genre her könnte ich mir alles mögliche vorstellen: Ein Quiz im Stil von You-Dont-know-Jack, ein Minigame im Stil von Tapper (Brauerei), ein kleines Adventure mit Cut-Szenen und Minigames im Stil von http://www.kirche-entdecken.de/lutherspiel/ ...   Vielleicht können wir am Sonntag mal darüber reden?

**Comment by t-karcher** [comment]
Idee: Eine relativ simple Brauerei-Wirtschaftssimulation im Stil von "Der Planer", die sich an der echten Geschichte von Schwalm-Bräu orientiert (https://www.schwalmbraeu.de/brauerei/historie/). Also mit Produktionsengpass von 1940-45, mit Umstellung von Pferdefuhrwerken auf LKW 1978, mehr Lagerfläche ab 1992 etc. als festen Ereignissen, auf die man dann irgendwie reagieren muss. Vielleicht auch mit Zwischenzielen, die erreicht werden müssen: So etwas wie
* Reduziere die Betriebskosten auf weniger als 2000 Reichsmark / Monat, um deine Familie ernähren zu können. (bis 1945)
* Baue 500.000 DM Vermögen auf, um die Lagerfläche zu vergrößern. (bis 1992)
* Investiere mindestens 800 Euro / Monat in die Forschung, um neue Biersorten einführen zu können. (bis 2008)
Und wird ein Ziel verfehlt, kann man entweder neu starten oder mit einer gespeicherten "Musterlösung" weiterspielen.

Hört sich erstmal sehr komplex an, aber ich denke da eher an was sehr Schlichtes. Ein rundenbasiertes Spiel mit wenigen Freiheitsgraden pro Runde ("Möchtest du dieses Jahr ... neue Mitarbeiter einstellen? ... Eine Kirmes ausrichten? ... Den Bierpreis erhöhen? ...")

**Comment by t-karcher** [comment]
Es muss nichtmal ein Videospiel sein! Idee: Wir könnten ein echtes (!) Kartenspiel im Stil von Anno Domini mit der Geschichte der Familie Haaß entwerfen. Also mit Ereignissen ("Rudolf Haaß wird geboren", "Die alten Stallungen der Brauerei weichen einer moderne Fahrzeughalle", "Zum Familientag treffen sich erstmals über 100 Nachfahren", ...), die zeitlich eingeordnet werden müssen.

**Comment by ChrisOKay** [comment]
entschuldige die späte Rückmeldung.
Die Idee der "stark begrenzten" Simulation finde ich sehr gut. Da müssen wir uns in Anbetracht unserer bisherigen Entwicklungshistorie nur harte Zwischenziele setzen, damit wir auch nur im Ansatz eine Chance der Fertigstellung haben.
Von der letzten des Kartenspiels bin ich nicht so überzeugt, weil der Wieder-Spielwert nicht hoch sein wird und wir damit effektiv nach einem ersten "schön" nur einen Haufen Müll & Produktionskosten erzeugen.

**Comment by t-karcher** [comment]
Meinetwegen gern!

Ich hätte auch schon eine Idee zum Design: Erinnerst du dich an das große Brauerei-Bild im Treppenhaus des Bräustübls? Ich fände es cool, wenn wir dieses und drei vier andere im selben Stil (z.B. das untenstehende) zu [Cinemagrammen](https://de.wikipedia.org/wiki/Cinemagramm) umgestalten könnten (mit animiertem Rauch, vorbeiziehenden Wolken o.ä.) und immer dann im Spiel zeigen könnten, wenn es um das entsprechende Thema (Ausbau des Gebäudes, Kauf von Ressourcen etc.) geht. Was meinst du?

Ist natürlich künstlerisch ziemlich anspruchsvoll, aber vielleicht können wir uns die Arbeit etwas erleichtern, indem wir Fotos mit Filtern in Bleistiftzeichnungen umwandeln und dann "nur noch" nachbearbeiten...

![image](https://user-images.githubusercontent.com/22897030/41604528-c59fa83c-73df-11e8-872b-118a15e2a29c.png)

**Comment by ChrisOKay** [comment]
Du und Deine cinemagramme. seit Jahren liegst du mir damit in den Ohren ;)
Können wir gerne machen, ist ja letztlich der "low-level-Animations-Rückzugsort", den wir schon aus Wing Commander Privateer in diversen Hangars kennen.

Als Simulation finde ich das von der Planung her etwas zu komplex, es sei denn du hast da schon eine konkretere Vision. Mir wären auf Anhieb kleine Minispiele (Reaktion, Puzzle, Rennen, etc...) lieber, die dann als kleine für sich gestellte Projekte abgeschlossen werden können.

**Comment by t-karcher** [comment]
Stimmt! Das hatte ich schon lange vergessen, aber der Vergleich mit Wing Commaner Privateer passt ganz gut.

Und nein: Ich habe leider auch noch keine richtige Vision für eine Simulation. Eine Art interaktive Geschichte mit Minispielen wären mir deshalb auch sehr recht.

**Comment by ChrisOKay** [comment]
Unity hat mir heute Nacht einen Newsletter mit ganz geeigneten Themen zur Spiele-Entwicklung gesendet.
Innerhalb von Drei Wochen ein [Minimalprodukt](http://app.response.unity3d.com/e/er?utm_campaign=engine_global_activation_Onboarding%20beginners%20flow&utm_content=17-onboarding-beginners-tutorial9&utm_medium=email&utm_source=Eloqua&s=795651218&lid=798&elqTrackId=2dfc8c9776ea4fab91a7f1c50e04174d&elq=05cb9c21e7e94543852dca3b02e7f837&elqaid=2034&elqat=1), welches spielbar ist aber nahezu keinen Extra-Content hat, zum anderen allgemeines zu [Milestones](https://unity3d.com/de/learn/tutorials/topics/developer-advice/setting-and-keeping-production-goals?playlist=17126&utm_campaign=engine_global_activation_Onboarding%20beginners%20flow&utm_content=17-onboarding-beginners-tutorial9&utm_medium=email&utm_source=Eloqua) (nichts neues, aber zumindest die 1-Wochen-Milestones finde ich für die Kürze der Zeit sehr gut)

**Comment by t-karcher** [comment]
Furchtbare Stimme, aber schöne Zusammenfassung der Gründe, warum unsere bisherigen Spiele fast alle gescheitert sind! :-)
Apropos Unity: Wie fit bist du denn mittlerweile darin? Dein Geburtstagsgeschenk sah ja schon recht vielversprechend aus! Meinst du, du könntest damit auch so ein Projekt realisieren? Ich selbst habe noch überhaupt keine Erfahrung damit, würde mich dann also auf alles andere außer der Programmierung (Konzept, Grafik, Sound, Test...) konzentrieren - was auch ok wäre. Oder wir bleiben für dieses Projekt noch bei Godot - da fühle ich mich einigermaßen fit drin. Was meinst du?

**Comment by ChrisOKay** [comment]
Ich finde beide Sprachen vom Konzept her recht ähnlich.
Was mir bei Unity auf jeden Fall mehr zusagt ist die breitere Basis, man findet schneller Antworten auf Fragen. Außerdem hat man mit Visual Studio eine ausgereiftere Entwicklungsumgebung.
"Fit" bin ich in keiner der beiden Sprachen, von best practices sicherlich weit entfernt. Aber wegen der breiteren Verteilung würde ich es gerne per Unity machen.

**Comment by t-karcher** [comment]
Ok - dann also mit Unity! Dann werd' ich mir das auch mal installieren und mich an ein paar Tutorials versuchen...
